theurgy.lic

```diff
@@ -744,11 +744,24 @@ class TheurgyActions
 
   def wine
     walk_to_altar_or_prayer_mat
-    DRCI.get_item_safe('wine', @theurgy_supply_container)
-    if @prayer_mat
-      fix_standing
-      bput("kneel #{@prayer_mat}", 'You humbly kneel')
+    if DRCI.get_item?("wine", @theurgy_supply_container)
```

Comment:
I'm a big fan of minimizing indentation. A good way to do that is using guard clauses. For example:

```suggestion
    return unless DRCI.get_item?("wine", @theurgy_supply_container)
```